### PR TITLE
Theme coding standards

### PR DESCRIPTION
**Pull request by rimi-itk**
https://jira.itkdev.dk/browse/ER-1317

Sets up [Prettier](https://prettier.io/) for checking theme coding standards and add GitHub Actions to run checks on pull requests.

### CODE REVIEW

File: .github/workflows/pr.yaml [hunk 1]
```diff
@@ -0,0 +1,79 @@
+on: pull_request
+name: Review
+jobs:
+  'assets-coding-standards-pratchett':
```
Comment (cableman) [inline .github/workflows/pr.yaml:4]: Please remove quotes around the job label (for all labels).

File: .github/workflows/pr.yaml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    name: Check assets coding standards (pratchett)
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: ['10']
```
Comment (cableman) [inline .github/workflows/pr.yaml:9]: You should update this to node 16 (which is the current LTS release). Please do this for all jobs.

File: .github/workflows/pr.yaml [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup node
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node }}
+      - run: |
+          yarn --cwd sites/all/themes/pratchett/ install
```
Comment (cableman) [inline .github/workflows/pr.yaml:17]: You should add cache to yarn to speed up tests (we have limit min. pr. month in actions with this free account).

For details see https://github.com/actions/cache 

```
- uses: actions/cache@v2
  with:
    path: '**/node_modules'
    key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
```

Please use this with all `install` of assets.

File: .github/workflows/pr.yaml (hunk 2 above)
Comment (cableman) [reply .github/workflows/pr.yaml:9]: Not required but one could also update the docker-compose setup and use that to ensure that the same version is used both in actions and in lokal development.

File: README.md [hunk 4]
```diff
@@ -170,7 +170,65 @@ docker-compose run --rm node bash -c "cd /app/sites/all/themes/wille/ && yarn in
...
+
+to apply the coding standards.
+
+## GitHub Actions
+
+We use [GitHub Actions](https://github.com/features/actions) to check theme
+coding standards whenever a pull request is made.
+
+Before making a pull request you can run the GitHub Actions locally to check for
+any problems:
```
Comment (cableman) [inline README.md:213]: Normally we will not recommend using act as it only partly supports what github actions can do. Its use more as a  test tool to try debug simple actions.

**Review by cableman — CHANGES_REQUESTED** [review]

File: .github/workflows/pr.yaml (hunk 1 above)
Comment (rimi-itk) [reply .github/workflows/pr.yaml]: Fixed.

File: .github/workflows/pr.yaml (hunk 2 above)
Comment (rimi-itk) [reply .github/workflows/pr.yaml:9]: Our assets cannot be built with node 16 (!). We'll fix this in another pull request.

File: .github/workflows/pr.yaml (hunk 3 above)
Comment (rimi-itk) [reply .github/workflows/pr.yaml:31]: Done.

**Review by cableman — APPROVED** [review]

**Pull request merged**